string command
---
float64[] state
